packages/styles/scss/patterns/FeaturedLink/_featuredlink-item.scss

```diff
@@ -0,0 +1,76 @@
+//
+// Copyright IBM Corp. 2016, 2018
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+@import '../../components/link-with-icon/link-with-icon';
+
+@mixin aspect-ratio($width, $height) {
+  position: relative;
+  &::before {
+    display: block;
+    content: '';
+    width: 100%;
+    padding-top: ($height / $width) * 100%;
+  }
+  > * {
+    position: absolute;
+    top: 16px;
+    left: 16px;
+    right: 0;
+    bottom: 0;
+  }
+}
```

Comment:
Is this mixin repeated from somewhere else? @annawen1 you created this one right? Perhaps we should start collecting some common mixins for ourselves...

Reply:
This was created by me jeff .... i think i took it from one of Raphaels PR.....

Reply:
@jeffchew  we have aspect ratio method in leadspace styles ... not sure wether we need a common mixin for these  ... need suggestions on this

Reply:
I guess I was thinking of this one, which looks different:

https://github.com/carbon-design-system/ibm-dotcom-library/blob/2d46e95274743e97c46404db895217b2e16c3544/packages/styles/scss/patterns/leadspace/_leadspace.scss#L13

For the one you created, I guess for now let's keep in your pattern, but let's give the mixin a name that's specific to this pattern since technically it would be available to anywhere after it's declared. Would help to provide a description of what it does too.